app_legacy/Helpers/database/game.php

```diff
@@ -191,8 +191,18 @@ function getGameMetadata(
     return $numAchievements;
 }
 
-function getGameAlternatives(int $gameID): array
+function getGameAlternatives(int $gameID, $sortBy = null): array
 {
+    $orderBy = match ($sortBy) {
```

Comment:
This code was adapted from line 37 where we sort achievements, with the SQL changed to match the query elements for game alternatives